Tests/NIOPosixTests/EventLoopTest+XCTest.swift

```diff
@@ -22,6 +22,7 @@ import XCTest
 /// Do NOT edit this file directly as it will be regenerated automatically when needed.
 ///
 
+@available(macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *)
```

Comment:
Were these two changes made by hand-editing?

Reply:
yes

Reply:
Ok, we can't make manual changes to these files (see the comment above). We'll either need to change the tool, or lift the availability guard from the type and move them into the method bodies using runtime checks.